ext/scheduled_communications/tests/phpunit/Civi/ScheduledCommunications/SendTest.php

```diff
@@ -29,6 +29,10 @@ public function setUp(): void {
     $this->mut = new \CiviMailUtils($this, TRUE);
   }
 
+  public function tearDown(): void {
+    \CRM_Utils_Time::resetTime();
```

Comment:
I think this will give the error about error handlers if you don't call parent::tearDown.

Reply:
Good catch 🎣

Reply:
Actually, if you've added reset to the parent, then don't need this at all. Or if you double-reset does time start going backwards?

Reply:
Oh wait, nevermind.

Reply:
Yea, not sure there's any good reason for this test to not have a parent... but whatever.